server/sync_watcher.go

```diff
@@ -272,13 +272,8 @@ func (watcher *SyncWatcher) processSyncWatch(ctx context.Context, path string) e
 				if response.Err != nil {
 					return response.Err
 				}
+				watcher.watchExtensionHandler(response)
 
-				watcher.queue.Add(job.NewJob(
```

Comment:
Anyway, this PR removes use of `watcher.queue`, but doesn't remove queue itself.